examples/simple/mig.tf

```diff
@@ -14,66 +14,48 @@
  * limitations under the License.
  */
 
-data "template_file" "group1-startup-script" {
-  template = "${file("${format("%s/nginx_upstream.sh.tpl", path.module)}")}"
-
-  vars {
-    UPSTREAM = "${module.gce-ilb.ip_address}"
-  }
+module "instance_template1" {
+  source             = "terraform-google-modules/vm/google//modules/instance_template"
```

Comment:
1.1.0 of the module has been released. All instances of the module should be pinned to `~> 1.1`.

Reply:
Fixed